CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ if(WIN32 AND MSVC_VERSION LESS 1800)
 endif()
 
 option(ENABLE_JSON_LOGGING "Adds additional JSON debug logging to the response" OFF)
+option(DEBUG_GEOMETRY "Enables an option to dump GeoJSON of the final routing graph" OFF)
```

Comment:
Can we move the debug output to a different PR? Should be easy to separate by splitting the commit with `git rebase -i`.